manifests/guestcluster/1.18/pvcsi.yaml

```diff
@@ -331,6 +332,7 @@ spec:
           - "--supervisor-fss-namespace=$(CSI_NAMESPACE)"
           - "--fss-name=internal-feature-states.csi.vsphere.vmware.com"
           - "--fss-namespace=$(CSI_NAMESPACE)"
+          - "--use-gocsi=false"
```

Comment:
In future, do we plan to default to this behavior and remove the option from the YAML like we plan to do for features that have been enabled in prior releases?